### PR TITLE
ci: Publish Flank-scripts to Github

### LINKED ISSUE #1565: Move publishing flank-scripts from bintray to other place
**Author the user story for this feature**

As a Developer, I want to upload flank-scripts to maven repository with direct link so I can download it when needed.

**Describe the solution**
Currently, we are uploading flank-scripts to bintray which will be closed at the end of April, so we should found another place to store flank-scripts binaries.

Please update also flank-scripts download task
```kotlin
tasks.register("download") {
    val sourceUrl = "https://dl.bintray.com/flank/maven/com/github/flank/$artifactID/$version/$artifactID-$version.jar"
    val destinationFile = Paths.get("bash", "flank-scripts.jar").toFile()
    ant.invokeMethod("get", mapOf("src" to sourceUrl, "dest" to destinationFile))
}
```

**Requirements**
1. Free
2. Easy to upload and download
3. Possibility to get a direct link to download jar

**Possible solutions**
- [Jitpack](https://jitpack.io/) 

**Describe alternatives considered**
As an alternative, we could upload flank-scripts to Github packages and get a link through Github API, but it needs one additional request with parsing output

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Fixes #1565

- [Flank scripts is released to Github Packages](https://github.com/Flank/flank/packages/628838)
- [Flank scripts is released to Github Releases](https://github.com/Flank/flank/releases/tag/flank-scripts-1.6.3)
- Above functions could be run using (`./gradlew flank-scripts:releaseFlankScripts`) ([Github CLI tool required](https://github.com/cli/cli))
- Download function works properly (`./gradlew :flank-scripts:download`) ([Github CLI tool required](https://github.com/cli/cli))
## Checklist

- [x] Publish to Github Packages
- [x] Publish to Github release
- [x] Update download function
- [x] JFrog deleted

### CODE REVIEW

File: flank-scripts/build.gradle.kts [hunk 1]
```diff
@@ -160,6 +151,17 @@ val checkIfVersionUpdated by tasks.registering(Exec::class) {
     }
 }
 
+val releaseFlankScripts by tasks.registering(Exec::class) {
+    //dependsOn(":flank-scripts:publish")
```
Comment (pawelpasterz) [inline flank-scripts/build.gradle.kts:155]:
```suggestion
```

**Review by pawelpasterz — APPROVED** [review]

File: flank-scripts/build.gradle.kts (hunk 1 above)
Comment (piotradamczyk5) [reply flank-scripts/build.gradle.kts:155]:
```suggestion
    dependsOn(":flank-scripts:publish")
```

Comment (piotradamczyk5) [reply flank-scripts/build.gradle.kts:155]: Actually, it should not be commented 😄

**Review by Sloox — APPROVED** [review]
Good work 🚀

**Pull request merged**